aptos-move/e2e-move-tests/src/tests/per_category_gas_limits.rs

```diff
@@ -46,10 +47,8 @@ fn io_limit_reached() {
     let acc = h.new_account_at(AccountAddress::from_hex_literal("0xbeef").unwrap());
     assert_success!(h.publish_package(&acc, &common::test_dir_path("execution_limit.data/test"),));
 
-    // Lower the max io gas to 10 units.
-    h.modify_gas_schedule(|gas_params| {
-        gas_params.txn.max_io_gas = Gas::new(10) * gas_params.txn.gas_unit_scaling_factor
-    });
+    // Lower the max io gas to lower than a single load_resource
+    h.modify_gas_schedule(|gas_params| gas_params.txn.max_io_gas = InternalGas::new(300_000 - 1));
```

Comment:
What is 300_000 - 1?

Reply:
300_000 is the gas charge for a single load_resource (see the comment above)